astropy/units/format/cds_lextab.py

```diff
@@ -11,7 +10,7 @@
 
 # cds_lextab.py. This file automatically created by PLY (version 3.11). Don't edit!
 _tabversion   = '3.10'
-_lextokens    = set(('CLOSE_BRACKET', 'CLOSE_PAREN', 'DIMENSIONLESS', 'DIVISION', 'OPEN_BRACKET', 'OPEN_PAREN', 'PRODUCT', 'SIGN', 'UFLOAT', 'UINT', 'UNIT', 'X'))
+_lextokens    = {'CLOSE_BRACKET', 'CLOSE_PAREN', 'DIMENSIONLESS', 'DIVISION', 'OPEN_BRACKET', 'OPEN_PAREN', 'PRODUCT', 'SIGN', 'UFLOAT', 'UINT', 'UNIT', 'X'}
```

Comment:
This is auto-generated, so makes little sense to include

Reply:
Ah. Good to know. Maybe I'll edit PLY some time to comply with py3.8+ preferred syntax

Reply:
Done.

Reply:
Rolling back PLY changes.